doc/install/manual_install.rst

```diff
@@ -14,20 +14,15 @@ need help installing Python, please refer to our :ref:`install-python` guide.
 Installing MNE-Python with all dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 If you use Anaconda, we suggest installing MNE-Python into its own ``conda`` environment.
-
-The dependency stack is large and may take a long time (several tens of
-minutes) to resolve on some systems via the default ``conda`` solver. We
-therefore highly recommend using the new `libmamba <https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community>`__
-solver instead, which is **much** faster. To permanently change to this solver,
-you can set ``CONDA_SOLVER=libmamba`` in your environment or run
-``conda config --set solver libmamba``. Below we just use ``--solver`` in each command.
+Ensure you are using a recent version of ``conda``, otherwise the dependency resolution
+may take extremely long or even fail!
```

Comment:
can we be specific here about which conda version is the recommended minimum version?  (presumably this is the first one where libmamba is the default solver and where the `channel::package` syntax is supported).

Reply:
the `channel::package` syntax was introduced in conda 4.4.0, released in Dec 2017:
https://docs.conda.io/projects/conda/en/4.6.0/release-notes.html#id71

Since it's so old, I will not mention this

The switch of the default solver to libmamba OTOH happened only a few months ago, so I will mention the specific version number

Reply:
@drammock I've addressed the issue of using a recent-enough `conda` version in [af16862](https://github.com/mne-tools/mne-python/pull/12362/commits/af168626b981f341de0fcf48c329940d6c8cc55a)